build/css/style.css

```diff
@@ -178,14 +181,18 @@ body {
   -webkit-box-align: center;
       -ms-flex-align: center;
           align-items: center;
+  line-height: 2rem;
   height: 500px; }
   @media (min-width: 600px) {
     .site-header {
       -webkit-box-orient: horizontal;
       -webkit-box-direction: normal;
           -ms-flex-direction: row;
               flex-direction: row;
-      height: 100%; } }
+      -webkit-box-pack: start;
+          -ms-flex-pack: start;
+              justify-content: flex-start;
+      height: 500px; } }
```

Comment:
Might want to leave this at 100% so it is more dynamic. If this height is done to vertically center, instead try using flexbox with flex direction column and align items center